planning/behavior_path_planner/autoware_behavior_path_start_planner_module/src/shift_pull_out.cpp

```diff
@@ -289,10 +289,10 @@ std::vector<PullOutPath> ShiftPullOut::calcPullOutPaths(
 
   bool has_non_shifted_path = false;
 
-  // if shift length is too short, add non sifted path
-  constexpr double MINIMUM_SHIFT_LENGTH = 0.01;
+  // if shift length is too short, add non shifted path
   const double shift_length = arc_position_start.distance;
-  const bool is_smaller_than_minimum = std::abs(shift_length) < MINIMUM_SHIFT_LENGTH;
+  const bool is_smaller_than_minimum =
```

Comment:
`th_distance_to_middle_of_the_road` and `MINIMUM_SHIFT_LENGTH` are parameters with different values. Is there an intention to replace it with th_distance_to_middle_of_the_road? 
While I think constant definitions are not preferable, I believe replacing it with parameters of different meanings would be inappropriate.

Reply:
Thank you @kyoichi-sugahara 

Initially I though both values presented the same thing: to check whether ego is on centerline or not. However, since after the changes no planner will be called if `isCurrentPoseOnEgoCenterline() == true` I will actually revert this change with the latest commit :+1: